libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -1293,6 +1297,12 @@ CK_DECLARE_FUNCTION( CK_RV, C_CloseSession )( CK_SESSION_HANDLE xSession )
             vSemaphoreDelete( pxSession->xVerifyMutex );
         }
 
+        if( NULL != pxSession->pxFindObjectLabel )
+        {
+            vPortFree( pxSession->pxFindObjectLabel );
+            pxSession->pxFindObjectLabel = NULL;
+        }
+
```

Comment:
the free libc free allows freeing of null variables (with no effect of course), not sure about vPortFree implementation 
if it is a requirements, then we don't need to check for != NULL

Reply:
Oh interesting, I'll try to remove this

Reply:
don't forget to create another commit to remove the if (ptr == null) before freeing